src/commands/trialApp/showTutorial.ts

```diff
@@ -15,6 +15,9 @@ export async function showTutorial(context: IActionContext): Promise<void> {
 
     if (trialAppNode) {
         const extensionId: string = 'redhat.vscode-didact';
+        if (!extensions.getExtension(extensionId)) {
+            window.showInformationMessage(localize('mustInstallDidact', 'The Didact extension must be installed to view the trial app tutorial.'));
```

Comment:
I prefer using this kind of error dialog:
https://github.com/microsoft/vscode-azurestaticwebapps/blob/master/src/commands/createHttpFunction.ts#L78-L80